service/core/action/meta/details.go

```diff
@@ -28,12 +30,16 @@ func details(w http.ResponseWriter, r *http.Request) {
 	}
 
 	metaType := r.URL.Query().Get("type")
+	omitScript := r.URL.Query().Get("omit_script")
+	if omitScript != "0" {
+		omitScript = "1"
+	}
```

Comment:
set omitScript to 1 as it will not be passed as query param

Reply:
Means I should not take it as query param and set it 1 in every requests?